src/components/Footer/Fotter.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { TodoStatus } from '../../types/Status';
+
+type Props = {
+  todos: Todo[];
+  status: TodoStatus;
+  onStatusChange: (arg: TodoStatus) => void;
+  clearCompletedTodos: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  status,
+  onStatusChange,
+  clearCompletedTodos,
+}) => {
+  const activeTodos = todos.filter(todo => !todo.completed);
+  const isAnyCompleted = todos.some(todo => todo.completed);
+  const filterOptions = Object.values(TodoStatus);
+
+  const capitalizeFirstLetter = (value: TodoStatus) => {
```

Comment:
Same - you can create this function outside of the component (or even in a separate helpers.ts file and import it here)